proposals/bring_your_own_argo_workflows/byoaw_test_plan.md

```
@@ -0,0 +1,628 @@
+### 2.2 Complex Pipeline Types
+
+| Test Case ID          | TC-PF-002                                                                                                                                                         |
+|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Test Case Summary** | Verify run of "Pipelines with artifacts" pipeline                                                                                                                 |
+| **Test Steps**        | 1. Execute pipeline - Pipelines with artifacts<br/>2. Verify each pipeline type executes correctly<br/>3. Validate artifacts, metadata, and custom configurations |
+| **Expected Results**  | - Pipeline execute successfully<br/>- Artifacts are produced to the right s3 location and are consumed correctly                                                  |
+
+| Test Case ID          | TC-PF-003                                                                                                                                                            |
+|-----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Test Case Summary** | Verify run of "Pipelines without artifacts" pipeline                                                                                                                 |
+| **Test Steps**        | 1. Execute Pipeline - Pipelines without artifacts<br/>2. Verify each pipeline type executes correctly<br/>3. Validate artifacts, metadata, and custom configurations |                                                                                                                                                                                   |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)